### PR TITLE
Bug 1330773 - Rename the fetch-allthethings and AllthethingsTransformerMixin task to something that mentions RunnableJobs

### PR DESCRIPTION
**Pull request by alisha17**
Bugzilla link: https://bugzilla.mozilla.org/show_bug.cgi?id=1330773

For the fetch_allthethings task: In treeherder/Procfile and /treeherder/bin/run_celery_worker, I wasn't really sure of what it does, so I didn't rename it for now.

Could you explain me the purpose of these files and hence, if it has to be renamed as well?

### CODE REVIEW

**Comment by alisha17** [comment]
The build failure me the answer ^ and I renamed in Procfile as well, but I would still like to know what it actually does?

File: treeherder/config/settings.py [hunk 1]
```diff
@@ -332,12 +332,12 @@ def server_supports_tls(url):
             "queue": "buildapi_4hr"
         }
     },
-    'fetch-allthethings-every-day': {
+    'fetch-runnablejobs-every-day': {
         'task': 'fetch-allthethings',
```
Comment (edmorley) [inline treeherder/config/settings.py]: This line need updating too

File: treeherder/etl/tasks/buildapi_tasks.py [hunk 2]
```diff
@@ -36,10 +36,10 @@ def fetch_buildapi_build4h():
     Builds4hJobsProcess().run()
 
 
-@task(name='fetch-allthethings', soft_time_limit=15 * 60)
-def fetch_allthethings():
+@task(name='fetch-runnablejobs', soft_time_limit=15 * 60)
+def fetch_runnablejobs():
     """
-    Fetches possible jobs from allthethings and load them
+    Fetches possible jobs from runnablejobs and load them
```
Comment (edmorley) [inline treeherder/etl/tasks/buildapi_tasks.py]: Let's use `allthethings.json` in this code comment instead

**Comment by alisha17** [comment]
@edmorley 
Updated this. Thanks :)

**Pull request merged**